geopandas/plotting.py

```diff
@@ -9,6 +9,10 @@ def plot_polygon(ax, poly, facecolor='red', edgecolor='black', alpha=0.5):
     """ Plot a single Polygon geometry """
     from descartes.patch import PolygonPatch
     a = np.asarray(poly.exterior)
+    if poly.has_z:
+        from shapely.geometry import Polygon
```

Comment:
Shapely is a hard dependency, so please move this to the top of the file.

The descartes import above is protected in this function to make that a soft dependency.